pkg/prm/backends/docker.go

```diff
@@ -19,3 +33,33 @@ func (*Docker) Exec(tool *prm.Tool, args []string) (prm.ToolExitCode, error) {
 	// TODO
 	return prm.FAILURE, nil
 }
+
+// Check to see if the Docker runtime is available:
+// if so, return true and info about Docker on this node;
+// if not, return false and the error message
+func (d *Docker) Status() prm.BackendStatus {
+	// The client does not error on creation if the background service is not running,
+	// but attempting to list the containers does.
+	dockerInfo, err := d.Client.ServerVersion(context.Background())
+	if err != nil {
+		message := fmt.Sprintf("%s", err)
```

Comment:
```suggestion
		message := err.Error()
```

Reply:
I would even suggest `fmt.Errorf` - I think some syntax checks might actually highlight this in `pdkgo`